application/controllers/admin/UserManagement.php

```diff
@@ -873,30 +882,19 @@ private function _createNewUser($aUser, $sendMail = true)
     }
 
     /**
-     * private method to store a change in a user
-     * return json to return to js frontend
+     * Update admin-user
      *
-     * @param array $aUser
-     * @param integer $userid
-     * @return string
+     * @param $aUser
+     * @return array|bool|mixed|null
      */
-    private function _editUser($aUser)
+    private function updateAdminUser($aUser)
     {
         $oUser = User::model()->findByPk($aUser['uid']);
-        $aReturnArray = ["sMessage" => gT('User successfully updated')];
-        if (isset($aUser['password'])) {
-            $rawPassword = $aUser['password'];
-            $aUser['password'] = password_hash($rawPassword, PASSWORD_DEFAULT);
-            $display_user_password_in_html = Yii::app()->getConfig("display_user_password_in_html");
-            $aReturnArray["sMessage"] .= $display_user_password_in_html ? "<br/>New password set: " . $rawPassword : '';
-        }
-
         $oUser->setAttributes($aUser);
         $oUser->modified = date('Y-m-d H:i:s');
 
-        $aUser = [];
         if ($oUser->save()) {
-            return $oUser->attributes();
+            return $oUser;
         }
         return false;
```

Comment:
No you must always return Model to get the errors

Reply:
ok